src/stream/src/task/mod.rs

```diff
@@ -147,7 +147,11 @@ impl SharedContext {
 
     #[inline]
     pub fn add_channel_pairs(&self, ids: UpDownActorIds, channels: ConsumableChannelPair) {
-        self.lock_channel_map().insert(ids, channels);
+        assert!(
```

Comment:
I remembered that there was a none assert before, but I later removed it due to another bug 🤣 Nice to see we've added it back.

Reply:
Okay, I will include the CI script change.